src/js/profile/wearable/config.js

```diff
@@ -39,6 +39,7 @@
 				ns.setConfig("popupFullSize", true, true);
 				ns.setConfig("scrollEndEffectArea", "screen", true);
 				ns.setConfig("enablePageScroll", true, true);
+				ns.setConfig("enableCircularScrollbar", true, true);
```

Comment:
is enableCircularScrollbar the name you defined? or is it fixed?

Reply:
Yes, It is defined.

Reply:
okay.